test/unit/abci/handlers/deliverTxHandlerFactory.spec.js

```diff
@@ -125,4 +174,53 @@ describe('deliverTxHandlerFactory', () => {
       expect(e).to.be.equal(error);
     }
   });
+
+  it('should set identity model if ST has IDENTITY_CREATE type', async () => {
+    await deliverTxHandler(identityRequest);
+
+    expect(identityRepositoryMock.store).to.be.calledWithExactly(identityFixture, dbTransaction);
```

Comment:
We should check that identity validate data is called. Basically, we need always to check all external calls (or non-calls) with arguments.